### PR TITLE
translate

### PR DESCRIPTION
**Pull request by rshulyar**

### CODE REVIEW

File: src/components/header/header.css [hunk 1]
```diff
@@ -112,5 +112,5 @@
     user-select: none;
     white-space: nowrap;
     /*width: 96px;*/
-    width: 190px; /*Костиль №2! Видалити після додавання всіх елементів header, i розкоментуваті закоментоване. Вносити правки за обставинами*/
+    width: 190px; /* Crutch №2! Remove after adding all the elements of the header, and comment out the annotated. Make adjustments to the circumstances */
```
Comment (rshulyar) [inline src/components/header/header.css:115]: ytrfuy

File: src/components/header/gmailLink/gmailLink.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './gmailLink.css';
+import {gmailLink} from './../../../config.js';
```
Comment (dobryanskyy) [inline src/components/header/gmailLink/gmailLink.js]: Please remove unneeded `js` ext.

File: src/components/header/header.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import GmailLink from './gmail-link';
+import GmailLink from './gmailLink';
 import './header.css';
 
 function Header() {
```
Comment (dobryanskyy) [inline src/components/header/header.js:5]: ` // todo: fill needed code`

File: src/config.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+let gmailLink = "https://mail.google.com/mail/&ogbl";
```
Comment (dobryanskyy) [inline src/config.js]:
```js
const config = {
    gmailLink: 'https://mail.google.com/mail/&ogbl',
};

export default config;

```

**Review by dobryanskyy — CHANGES_REQUESTED** [review]

File: src/config.js (hunk 4 above)
Comment (dobryanskyy) [reply src/config.js]: https://gmail.com

**Pull request merged**